readthedocs_ext/readthedocs.py

```diff
@@ -22,7 +21,62 @@
 ]
 
 
+def finalize_media(app):
+    """
+    Point media files at our media server.
+    """
```

Comment:
We're not using pep257 on the extension yet, though we should. This single line comment should be on one line